src/components/AddMovie/AddMovie.tsx

```diff
@@ -1,13 +1,27 @@
 import BasicForm from '../BasicForm/BasicForm'
-import CloseButton from '../CloseButton/CloseButton'
 import style from './Addmovie.module.scss'
+import Button from '../Button/Button'
+import SuccessNotification from '../SuccessNotification/SuccessNotification'
+import { useState } from 'react'
+import Portal from '../Portal/Portal'
 
-const AddMovie = () => (
-  <div className={style.add_movie}>
-    <CloseButton />
-    <h2 className={style.topic}>ADD MOVIE</h2>
-    <BasicForm />
-  </div>
-)
+interface addMovieProps {
+  onClick: () => void
+}
+const AddMovie = ({onClick}:addMovieProps) => {
+  const [showModal, setShowModal] = useState(false)
+  return (
+      <div className={style.add_movie}>
+        <Button classname={'close_button'} onClick={onClick}/>
+        <h2 className={style.topic}>ADD MOVIE</h2>
+        <BasicForm />
+        {showModal && (
+          <Portal>
+            <SuccessNotification onClick={() => setShowModal(false)} />
+          </Portal>
+        )}
```

Comment:
You need to define the Form component itself as Portal. so you'd put the Form inside Portal and the form itself would be visible within the portal root. Here `BasicForm` is rendered inside AddMovie component